AI/MIGraphX/Quantization/Running Quantized ResNet50 via MIGraphX.md

```diff
@@ -0,0 +1,102 @@
+
+
+# Running quantized ResNet50 via MIGraphX
+
+## Summary
+This example walks through the dynamo Post Training Quantization (PTQ) workflow for running a quantized model using torch_migraphx.
+
+## Prerequisites
+
+- You must follow the installation instructions for the torch_migraphx library in [Readme.md](https://github.com/Rmalavally/rocm-examples/blob/develop/AI/MIGraphX/Quantization/Readme.md) before using this example.
```

Comment:
Change to the correct org

Reply:
@causten The content in Readme.md accompanying the example is different from the Readme in the repo. 

We have the following link at the end of the file for installation instructions,

>Refer to the [Torch_MIGraphX](https://github.com/ROCmSoftwarePlatform/torch_migraphx/blob/master/README.md) page for Docker and source installation instructions.

@shivadbhavsar Please confirm.

Reply:
@causten Let me know if you have any concerns. The Readme with the example provides contextual details about the example. If you are ok with this, can we merge the PR? Please advise.